docs/source/tutorial.rst

```diff
@@ -171,7 +171,43 @@ Let's say your proxy certificate is ``cert.pem``, then run the exceute request a
 
   $ curl --cert cert.pem --key cert.pem -k "http://localhost:8000/ows/proxy/emu?service=WPS&version=1.0.0request=Execute&identifier=hello&DataInputs=name=tux"
 
+Keycloak example
+================
+
+Set-up a demo Keycloak service using an Ansible `playbook <https://github.com/bird-house/ansible-keycloak-playbook>`_.
+
+The keycloak service is available at (``username=admin``, ``password=admin``):
+http://localhost:8080/auth/
+
+You need to copy the public key of your Keycloak realm to the twitcher configuration (see screenshot):
+
+.. image:: _images/keycloak-realm-public-key.png
+
+Update your twitcher configuration in ``development.ini``:
+
+.. code-block:: ini
+
+  twitcher.token.type = keycloak_token
+  keycloak.token.secret = public_key_copied_from_keycloak
+
+Start the twitcher service and register the Emu_ WPS:
+
+.. code-block:: console
+
+  $ twitcherctl -k --username demo --password demo register --name emu http://localhost:5000/wps
+
+Try the demo notebook to access a token from the keycloak and execute a WPS process.
+
+Use ``client_id=demo`` and copy the client secret from Keycloak in `Clients/demo/Credentials/Secret` (see screenshot).
+
+.. image:: _images/keycloak-client-secret.png
+
+.. toctree::
+   :maxdepth: 1
+
+   notebooks/twitcher-keycloak-demo
```

Comment:
That section did not display correctly, missing `.ipynb` extension?

![2019-11-13-135928_1005x877_scrot](https://user-images.githubusercontent.com/11966697/68794959-fc9a2480-061d-11ea-9c57-0f5b4d49419b.png)

Reply:
Needs `nbsphinx` to render. I have updated the readthedocs config ... hopefully this works.